packages/utils/src/lib/reports/log-stdout-summary.ts

```diff
@@ -65,11 +64,19 @@ function logPlugins(report: ScoredReport): void {
   });
 }
 
-function logCategories({ categories, plugins }: ScoredReport): void {
+const passIcon = bold(green('✓'));
+const failIcon = bold(red('✗'));
+export function logCategories({ categories, plugins }: ScoredReport): void {
   const hAlign = (idx: number) => (idx === 0 ? 'left' : 'right');
-  const rows = categories.map(({ title, score, refs }) => [
+
+  const rows = categories.map(({ title, score, refs, isBinary }) => [
     title,
-    applyScoreColor({ score }),
+    // @TODO refactor `isBinary: boolean` to `targetScore: number` #713
+    `${targetScoreIcon(score, isBinary === true ? 1 : undefined, {
+      passIcon,
+      failIcon,
+      postfix: ' ',
+    })}${applyScoreColor({ score })}`,
```

Comment:
Similar here, multiline function calls in template strings are quite hard to read, so I would extract it to a helper for the stdout reports. Something like this:

```ts
function binaryIconPrefix(score: number, isBinary: boolean | undefined): string {
  return targetScoreIcon(score, isBinary ? 1 : undefined, {
    passIcon: bold(green('✓')),
    failIcon: bold(red('✗')),
    postfix: ' ',
  });
}

```

```suggestion
    `${binaryIconPrefix(score, isBinary)}${applyScoreColor({ score })}`,
```